extensions/amp-analytics/0.1/visibility-manager-for-mapp.js

```diff
@@ -107,7 +107,8 @@ export class VisibilityManagerForMApp extends VisibilityManager {
   getRootLayoutBox() {
     // By the time `#getRootLayoutBox` is called, it is guaranteed that
     // onVisibilityChangeHandler has been called at least once
-    return devAssert(this.intersectionRect_);
+    return /** @type {!../../../src/layout-rect.LayoutRectDef} */ (
+      devAssert(this.intersectionRect_));
```

Comment:
@jridgewell Let me know if you can think of a fix for `devAssert` returning a `null|{T}`. Until then, we're going to have to typecast its return value, or use alternatives like `dev().assertElement()`.

Reply:
@kristoferbaxter The size increases pointed out by you and @jridgewell are coming from changes of this sort. The problem is that `devAssert` returns the type `null|{T}`, which breaks when a non-null value is expected. Here, I was able to fix it with a cast. In other places, I had to use `dev().assertElement` because casting to `Element` is forbidden by presubmit checks.

Let me know if you can think of a fix for `devAssert`, in which case I can revert most of the changes that cause a size increase.